packit/api.py

```diff
@@ -698,6 +697,7 @@ def sync_release(
         sync_default_files: bool = True,
         local_pr_branch_suffix: str = "update",
         mark_commit_origin: bool = False,
+        allow_missing_upstream_specfile: bool = False,
```

Comment:
I'll be honest - I am not a fan of this option name: especially the part "if it's missing, use the one from downstream". That's surprising to me, hence I'd suggest naming it "use_downstream_specfile". It would clearly indicate that the upstream specfile is ignored and the downstream one is used.

but since this is our code and it's not leaking to users and is well documented, I'm okay as it is

Reply:
resolved via chat: I used the name since previously the implementation was copying the file and not setting the specfile object and therefore I wanted this to work for the packages where there is the upstream specfile present (e.g.our packages). With the current implementation, it makes sense to rename this as Tomas suggested, so I just changed it.